src/server/node_types.rs

```diff
@@ -112,6 +112,15 @@ impl<T: Attributes> fmt::Debug for Node<T> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct DataTypeNode {
+    pub requested_new_node_id: Option<ua::NodeId>,
+    pub parent_node_id: ua::NodeId,
+    pub reference_type_id: ua::NodeId,
+    pub browse_name: ua::QualifiedName,
+    pub attributes: ua::DataTypeAttributes,
+}
```

Comment:
Let's use the same order of declarations in this file as in `ua/data_types/node_class.rs`, i.e.

- Object
- Variable
- Method
- …
- DataType